static/js/tools/stat_var/provenance.tsx

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Component } from "react";
+import { formatNumber } from "../../i18n/i18n";
+import { ProvenanceSummary } from "../../shared/types";
+
+interface ProvenancePropType {
+  provId: string;
+  summary: ProvenanceSummary;
+}
+
+class Provenance extends Component<ProvenancePropType, unknown> {
+  render(): JSX.Element {
+    return (
+      <div className="card p-0">
+        <div className="provenance-summary">
+          <h4>{this.props.summary.importName}</h4>
+          <div className="detail-text">
+            <div>
```

Comment:
it's nice to use semantic tags when possible (rather than plain divs). in this case, it could be a ul / li without the bullets. (besides the semantic annotation, it also helps screen readers, gives a target to style, etc)

Reply:
ah thanks this makes sense! (I didn't realize about the screen readers) updated to use ul/li